main.cc

```diff
@@ -933,10 +934,19 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             }
             view_hints_dir_initializer.ensure_created_and_verified().get();
 
+            static sharded<wasm::udf_cache> udf_cache;
+            supervisor::notify("starting udf cache");
```

Comment:
This name is a little too generic - let's emphasize that it's for wasm, because it won't cache any other instances. Also, this service should only be started if udfs and wasm are enabled in the first place - otherwise it just takes resources.

Reply:
Ok, I'll change the service name to somehing like `wasm_udf_cache`, I'll also add a condition for starting the service.